bip-0328.mediawiki

```diff
@@ -5,7 +5,7 @@
   Author: Ava Chow <me@achow101.com>
   Comments-Summary: No comments yet.
   Comments-URI: https://github.com/bitcoin/bips/wiki/Comments:BIP-0328
-  Status: Draft
+  Status: Proposed
```

Comment:
In commit 4a1837016a57608795f936da98ff91ce316dc800, also update the README status and formatting (Needs +> |- style="background-color: #ffffcf")

Reply:
Fixed this